bench/add-property.js

```diff
@@ -0,0 +1,97 @@
+const Benchmark = require('benchmark')
+const suite = new Benchmark.Suite;
+const { eventToMdTable, H2, createTableHeader } = require('../markdown')
+
+const tableHeader = createTableHeader([
+  'name',
+  'ops/sec',
+  'samples'
+])
+
+const cachedPropertyWritable = {
+  get() {
+    return 'Hello';
+  },
+  enumerable: true,
+  configurable: true,
+};
+
+const cachedPropertyDescriptor = {
+  get() {
+    return 'Hello';
+  },
+  enumerable: true,
+  configurable: true,
+};
+
+suite.add('Directly in the object', function () {
+  const data = { test: 'Hello' }
+})
+.add('Using dot notation', function () {
+  const data = { }
+  
+  data.test = 'Hello';
+})
+.add('Using dot notation with undefined value', function () {
+  const data = { test: undefined }
+  
+  data.test = 'Hello';
+})
+.add('Using getter', function () {
+  const data = { get test() { return 'Hello' } }
+})
+.add('Using define property (writable)', function () {
+  const data = { }
+  
+  Object.defineProperty(data, 'test', {
+    value: 'Hello',
+    writable: true,
+    enumerable: true,
+    configurable: true,
+  })
+})
+.add('Using define property cached (writable)', function () {
+  const data = { }
+  
+  Object.defineProperty(data, 'test', cachedPropertyWritable);
+})
```

Comment:
I added because I saw someone doing this with EventTarget to optimize so I want to make sure we could save some time doing this, I think this benchmark is worth having as well.

Reply:
But this wouldn't be a fair comparisson

Reply:
I will remove it for now but it will be nice to have the op/s to measure the difference between defining properties with the options being created at runtime and the options being cached.

Maybe `object-define-property.js`? But sounds a little bit redundant with this file.